en/docs/develop/hosting-authentication-endpoint-on-a-different-server.md

```diff
@@ -1,80 +1,39 @@
 # Hosting Authentication Endpoint on a Different Server
 
-The authenticationendpoint contains the authentication URLs used in
-authentication flow. You can either host
-the authenticationendpoint webapp on the WSO2 Identity Server, or choose
-to host it on a separate server. You may want to host it separately for
-the purpose of having custom theming and branding. This section
-describes how you can host the authentication endpoint on a different
-server outside the WSO2 Identity Server  (e.g., in a different Tomcat
-Server).
+The authenticationendpoint contains the authentication URLs used in authentication flow. You can either host the authenticationendpoint webapp on the WSO2 Identity Server, or choose to host it on a separate server. You may want to host it separately for the purpose of having custom theming and branding. 
```

Comment:
`You can either host the authenticationendpoint webapp on the WSO2 Identity Server` >> You can use the defualt authenticationendpoint webapp hosted on the WSO2 Identity Server itself, or choose to host...